app/views/confirmation.scala.html

```diff
@@ -81,9 +81,10 @@ <h2 class="govuk-heading-m govuk-!-margin-bottom-0">@messages("confirmation.subm
         </section>
     }
 
-    <p class="govuk-body">@messages("feedback.contact") <a class="govuk-link" href="mailto:@messages("feedback.email")?subject=@submissionId">@messages("feedback.email")</a></p>
-    <p class="govuk-body">@messages("feedback.contact2")</p>
-
+    @if(reportStatus.flatMap(_.status) == Some(Done.value) || reportStatus.flatMap(_.status) == Some(Submitted.value)) {
```

Comment:
I don't know what is better, your or my?
`reportStatus.map(status => status == Done.value || status == Submitted.value).getOrElse(false)`
At least you get rid of `Some`

Reply:
`status` is itself an option so you would still need the `Some`, with your method